### ISSUE TITLE
[Feature] Add posibility of Appointments with payment (Stripe)

### ISSUE THREAD

**Issue by ssergio-ll** [SergeyMosin/Appointments#369]
It would be very interesting to add the option for certain appointments to be paid for the user. That is, the client pays a reservation and once this is done it is confirmed.

**Comment by SergeyMosin** [comment]
Duplicate of #172 
In general there needs to be a separate NC app like "Nextcloud Stripe Integration" for example that other apps like Appointments can use via an internal API.
